spec/models/role_spec.rb

```diff
@@ -17,35 +17,35 @@
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 #++
 
-require File.expand_path(File.dirname(__FILE__) + "/../spec_helper")
+require File.expand_path(File.dirname(__FILE__) + '/../spec_helper')
 
-describe Role, :type => :model do
-  describe "class methods" do
+describe Role, type: :model do
+  describe 'class methods' do
     describe '#givable' do
-      before (:each) do
-        #this should not be necessary once Role (in a membership) and GlobalRole have
-        #a common ancestor class, e.g. Role (a new one)
-        @mem_role1 = Role.create :name => "mem_role", :permissions => []
-        @builtin_role1 = Role.new :name => "builtin_role1",  :permissions => []
+      before do
+        # this should not be necessary once Role (in a membership) and GlobalRole have
+        # a common ancestor class, e.g. Role (a new one)
+        @mem_role1 = Role.create name: 'mem_role', permissions: []
+        @builtin_role1 = Role.new name: 'builtin_role1', permissions: []
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.